worldedit-core/src/main/java/com/fastasyncworldedit/core/history/changeset/FaweStreamChangeSet.java

```diff
@@ -35,11 +36,18 @@
 public abstract class FaweStreamChangeSet extends AbstractChangeSet {
 
     public static final int HEADER_SIZE = 9;
-    private static final int version = 1;
+    private static final int VERSION = 2;
+    // equivalent to Short#MIN_VALUE three times
+    private static final byte[] MAGIC_NEW_RELATIVE = new byte[]{(byte) 128, 0, (byte) 128, 0, (byte) 128, 0};
```

Comment:
might be worth to not that this is big endian

Reply:
The initial implementation was in fact wrong, so adjusted to match how it would actually be stored and added a descriptor for it